signer.js

```diff
@@ -42,6 +72,42 @@ class SignatureSigner {
   }
 }
 
+/**
+ * This class is used for signing certificates using elliptic library
+ */
+class EllipticSignatureSigner extends SignatureSigner {
+  /**
+   * @param {string} pem
+   */
+  constructor(pem) {
+    super(pem);
+  }
+
+  /**
+   * This method is used for signing the content
+   * @param {Uint8Array} content
+   * @return {Uint8Array}
+   */
+  sign(content) {
+    try {
+      const base64 = this.getBase64PemString();
+      const {prvKeyHex} = jsrsasign.KEYUTIL.getKey(
+          jsrsasign.b64utohex(base64), null, 'pkcs5prv');
+      const EC = elliptic.ec;
+      const ecdsaCurve = elliptic.curves['p256'];
+      const ecdsa = new EC(ecdsaCurve);
+      const signKey = ecdsa.keyFromPrivate(prvKeyHex, 'hex');
```

Comment:
OK, ...
I am afraid elliptic might provide functions for parsing PKCS#5 PEM files.
We still need jsrsasign for doing that.

Reply:
@scalarindetail Yeah, I'm not sure if elliptic is enough. 
Do you know what elliptic can do and what jsrsasign can do ?

Reply:
As far as I checked, elliptic library does not have parsing PEM files. I think will still need to use jsrsasign just as utilities.

Reply:
@feeblefakie 
We can simply split the singer's goci into two parts
1. PEM file (private key encoded in PKCS#5 format) parsing
2. Generate content digest by SHA256
3. Signing digest by ECDSA

jsrsasign library can conduct these 3 steps by its own.
elliptic library seems supporting 3 only.

In the case that we elliptic to sign, we still need to use jsrsasign library to do 1, and crypto library to do 2.  crypto library is fine because it is built-in both on node.js and browser runtime. We just need to keep jsrsasign in the dependency.